library/psa_crypto_mac.c

```diff
@@ -30,18 +30,7 @@
 #include <mbedtls/error.h>
 #include <string.h>
 
```

Comment:
In my opinion there should be a naming convention for the local functions. All of them should use the `_internal` suffix or none of them.

Reply:
Historically, the `xxx_internal` functions are functions for which `xxx` was part of the API, and we needed part of its job to be a separate function. There's no general rule for what `_internal` indicates (the split in functionality is different for different `xxx`) and it would be good to have a more informative naming convention. However, I think this is out of scope of the current pull request. There are a number of pending follow-ups for this PR so I'd prefer for it to be merged as soon as possible, and to do naming improvements in a follow-up.